app/representers/api/paged_collection_representer.rb

```diff
@@ -1,63 +1,5 @@
-# encoding: utf-8
+require 'hal_api/representer/collection_paging'
 
 class Api::PagedCollectionRepresenter < Api::BaseRepresenter
```

Comment:
Hmm... doesn't the HalApi already include the equivalent of this representer?

Reply:
similar, but wrong app specific curies, so need to define our own - the `Api::BaseRepresenter` provides that

Reply:
makes sense.